README.md

```diff
@@ -82,55 +76,30 @@ Atheris is fully supported by [OSS-Fuzz](https://github.com/google/oss-fuzz), Go
 
 ## API
 
-### Main Interface
-
-The `atheris` module provides two key functions: `Setup()` and `Fuzz()`.
+The `atheris` module provides three key functions: `Instrument()`, `Setup()` and `Fuzz()`.
 
-In your source file, define a fuzzer entry point function, and pass it to `atheris.Setup()`, along with the fuzzer's arguments (typically `sys.argv`). Finally, call `atheris.Fuzz()` to start fuzzing. Here's an example:
+In your source file, when you import your target library make sure that this happens inside a `with atheris.Instrument():`-block.
```

Comment:
This is unusual, so provide an example. Also, replace "when you import your target library make sure that this happens inside" with "Import all libraries you wish to fuzz inside".